### ISSUE TITLE
Welcome message in REPL

### ISSUE THREAD

**Issue by rexim** [morganey-lang/Morganey#253]
REPL should print a standard splash with Welcome message and version of Morganey.

**Comment by keddelzz** [comment]
For a idea how the message could be, here's the welcome message of scala's repl.

```
Welcome to Scala 2.11.8 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_101).
Type in expressions for evaluation. Or try :help.
```

See also the position of this message in the scala [sources](https://github.com/scala/scala/blob/05016d9035ab9b1c866bd9f12fdd0491f1ea0cbb/src/compiler/scala/tools/nsc/Properties.scala#L20).

**Comment by keddelzz** [comment]
To print some welcome message like `Welcome to Morganey REPL (<version>)` we need the current version  of Morganey.

One way to achieve this is with the sbt plugin [sbt-buildinfo](https://github.com/sbt/sbt-buildinfo). Using it you can generate sources (containing version numbers) from the sbt build definition.

---  
There are of course multiple ways to implement this, but I just scrolled through the issues and remembered [sbt-buildinfo](https://github.com/sbt/sbt-buildinfo), so i wanted to leave a note.

**Comment by keddelzz** [comment]
I just saw, that we're already using sbt-buildinfo (see [here](https://github.com/morganey-lang/Morganey/blob/9a48f6ce99474007576b9345ba9791c631bd6fa2/build.sbt#L18) and [here](https://github.com/morganey-lang/Morganey/blob/9a48f6ce99474007576b9345ba9791c631bd6fa2/project/plugins.sbt#L7)).

**Comment by rexim** [comment]
Right, we use it to properly compose a name of the fatjar to run it for Funtional Testing, if I remember correctly.
